src/App.tsx

```diff
@@ -1,73 +1,39 @@
 import React from 'react';
```

Comment:
you can import hooks right away so you can use them right away instead of `React.hookName`
```suggestion
import React, { useState, useMemo } from 'react';
```